src/main/java/org/apache/commons/pool2/impl/GenericKeyedObjectPoolMXBean.java

```diff
@@ -91,6 +91,12 @@
      */
     boolean getLifo();
 
+    /**
```

Comment:
You MUST use default methods when adding to an interface in order to preserve binary compatibility.

Reply:
Could easily do, but the class had already a comment that this interface is subject to change:

 * NOTE: This interface exists only to define those attributes and methods that
 *       will be made available via JMX. It must not be implemented by clients
 *       as it is subject to change between major, minor and patch version
 *       releases of commons pool. Clients that implement this interface may
 *       not, therefore, be able to upgrade to a new minor or patch release
 *       without requiring code changes.

Reply:
Sure, but I bet this comment was written pre-Java 8 so I think you can drop that comment when you add default methods.

Reply:
Added default methods to the newly added methods